src/ontology/fbbt-edit.obo

```diff
@@ -44756,6 +44769,7 @@ intersection_of: part_of FBbt:00003022 ! adult metathoracic segment
 id: FBbt:00004409
 name: metathoracic papilla
 def: "One of a pair of campaniform sensilla of the haltere sclerite. It is an elliptical sensillum with a high profile and socket (type 5). The two sensilla are fused to each other by the cuticle of their sockets. It is around 5 micrometers in diameter." [FlyBase:FBrf0031004, FlyBase:FBrf0038304]
+comment: A recent comprehensive study of the campaniform sensilla (Dinges et al., 2020) found those sensilla at the base of the haltere, on the metanotum in close proximity to the proximal scabellum, instead of on the dorsal haltere surface. [FlyBase:FBrf0247616]
```

Comment:
Good point, I had not checked our definition of `haltere sclerite`. Dinges et al. (2020) say that the metathoracic papilla are on the *metanotum* and make no mention of the haltere sclerite. But on their micrographs, the location of the sensilla does seem to correspond with the haltere sclerite and *not* the metanotum.

I think the discrepancy may be due to the fact that Dinges et al. may simply make no distinction between the metanotum and the haltere sclerite (i.e. they consider the "haltere sclerite", where the haltere is attached, to be part of the metanotum instead of being a distinct sclerite -- the haltere sclerite being in the continuity of the metanotum).

So I believe our current definition was in fact correct. I will remove the comment (but add the Dinges et al. reference to the definition, as it may be more useful than references to decades-old paper).